code/game/antagonist/outsider/ert.dm

```diff
@@ -29,6 +29,7 @@ var/datum/antagonist/ert/ert
 	player.current << "You should first gear up and discuss a plan with your team. More members may be joining, don't move out before you're ready."
 
 /datum/antagonist/ert/equip(var/mob/living/carbon/human/player)
+	var/mob/living/M = player
```

Comment:
I'm not sure, but from my experience, it doesn't seem to work unless you define that. I'm not really focusing on this project at the moment, anyway - MFCS is my main project.